event-buffer/event-buffer-core/src/main/java/uk/gov/justice/services/event/buffer/core/service/BufferInitialisationStrategyProducer.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.services.event.buffer.core.service;
+
+
+import uk.gov.justice.services.common.configuration.Value;
+import uk.gov.justice.services.event.buffer.core.repository.streamstatus.StreamStatusJdbcRepository;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Produces;
+import javax.inject.Inject;
+
+@ApplicationScoped
+public class BufferInitialisationStrategyProducer {
+
+    private static final String TRUE = "true";
+    @Inject
+    @Value(key = "postgres.db", defaultValue = "true")
+    String usePostgresStrategy;
```

Comment:
Yes, the Wildfly config would need to change - I think selecting a non-standard strategy should be an explicit thing. This would need to be communicated to devops.